woocommerce/woocommerce
# Render password meter in place

### All Submissions:






### Changes proposed in this Pull Request:



Hides and reveals the password meter instead of removing it from the DOM. This allows themes to manually place the meter element and have it render in a consistent location.

Closes #19970.

### How to test the changes in this Pull Request:

1. Ensure registration is enable in WC>Settings>Accounts
2. Go to my account page, logged out
3. Use the password field. Strength meter appears.
4. Remove text from password field. Meter is hidden.
5. Check DOM. Meter is there as a hidden element.

@claudiulodro could be 3.4 but I'll leave to your best judgement.
